src/locales/ar/messages.json

```diff
@@ -26,5 +26,11 @@
   "signUp": "إنشاء حساب",
   "name": "الأسم",
   "email": "البريد الشخصي",
-  "passConfirm": "تأكيد كلمة المرور"
+  "passConfirm": "تأكيد كلمة المرور",
+  "bio": "السيرة الذاتية",
+  "linkedin": " Linkedin حسابك بموقع",
+  "github": "Github حسابك بموقع",
+  "twitter": "Twitter حسابك بموقع",
```

Comment:
Those 3 messages are the same, the only difference is the social network name. I think it is a good place to make use of [interpolation](https://www.i18next.com/translation-function/interpolation)
We can use one message and dynamically add the social network name to it.

Similar Example: 
```
"socialSignIn": "{{provider}} تسجيل الدخول باستخدام "
```